### ISSUE TITLE
labeled landmarks have empty columns in .csv output file

### ISSUE THREAD

**Issue by imgross** [DeepLabCut/DeepLabCut#872]
Hi there,

**My Operating system and DeepLabCut version**
Windows 10 with an Anaconda Env, & DeepLabCut 2.2b7, wx Python 4.0.7.post2, using maDLC working witht the Project Manager GUI

**Describe the problem**
1) I have a question regarding the .h5 output file that one gets after analyzing videos. I have converted the .h5 files I get into .csv files to further analyze results with Matlab. And in my tracking I have 3 stable landmarks that are present in every frame (as uniquebodyparts in the config.yaml file) . But those columns do not have any values in my .csv file, even though I see them labeled in the video that was created in the analyze video section. Do you know how I can access them?

2) The hot keys (arrows) for moving through frames in the tracklet GUI do not work for me. 

Thanks,
Isa

**Comment by MMathisLab** [comment]
Without more information, it's not clear to me what h5 to csv you are converting; but you can always use the h5 file in matlab, and we really recommend this - much better format for metadata, etc in the long term:

https://www.mathworks.com/help/matlab/ref/h5read.html

**Comment by imgross** [comment]
The h5 file I am converting is the one that was generated after I opened the tracklets GUI and hit save. So that one did neither have had any corrections from the tracklet GUI on it, nor any filtering. And when I load that h5 into Matlab the problem is, that the column names are absent, which is why I do not know to which bodyparts/individuals the values belong to. But when opening that file with python I do get the column names and there I see that the tracks for the landmarks are empty. This observation is then reproduced when converting that h5 into csv.

**Comment by MMathisLab** [comment]
HI Isa, then the issue is how the file is being read; if the info is there in Python (which I do see), then it's a matter of being sure you are indexing the file correctly. @jeylau perhaps can confirm, but the h5->csv function within DLC is working for maDLC.

**Comment by jeylau** [comment]
@imgross, I have just given a try and dumping to h5/converting to csv is functional here. If you re-load that h5 file in the Tracklet GUI, are the uniquebodyparts still visible or have they disappeared? How exactly did you convert the file?

**Comment by imgross** [comment]
Hi @jeylau , I think the problem was due to the way I chose the parameters for the tracklet GUI. 

If for example I set:
min_swap_len = 0
min_tracklet_len = 1
max_gap = 5
I did not see the problem. Here, the landmarks and other single bodyparts were visible in the GUI and in the h5 output file.

But when setting the params as following:
min_swap_len = 0
min_tracklet_len = 2
max_gap = 0
I only could see the multianimal labels in the tracklet GUI and similarly in the h5 output file.

Thank you,
Isa
